src/algo/dominators.rs

```diff
@@ -147,7 +149,7 @@ where
 
 /// The undefined dominator sentinel, for when we have not yet discovered a
 /// node's dominator.
-const UNDEFINED: usize = ::std::usize::MAX;
+const UNDEFINED: usize = core::usize::MAX;
```

Comment:
Using `std::` or `core::` is deprecated with MAX:
https://doc.rust-lang.org/core/usize/constant.MAX.html

Maybe there should be just changed to `usize::MAX` as the doc says. (as you're making changes in it).

Reply:
Nice catch!
Done